components/team-info/sections/TeamSection.vue

```diff
@@ -1,16 +1,16 @@
 <template>
-  <div id="team" class="overflow-hidden flex flex-col items-center space-y-5">
-    <h1 class="text-3xl font-bold mb-0">Our Team</h1>
-    <div class="flex space-x-10">
+  <div id="team" class="overflow-hidden md:flex flex-col items-center space-y-10">
+    <h1 class="text-3xl text-center font-bold mb-0">Our Team</h1>
+    <div class="flex flex-col items-center space-y-10 md:flex-row md:space-y-0 md:space-x-10">
       <team-member-component name="Viktor" role="Role" />
```

Comment:
Can you please also make the heads bigger? And if you want to you can also fill roles as they should be 
![image](https://github.com/fiit-tp7-2023/frontend/assets/64609060/06db0041-cb04-4e35-ad43-efeef288a7ad)